docs/jwst/cube_build/arguments.rst

```diff
@@ -98,3 +99,10 @@ The arguments related to region of interest and how the fluxes are combined toge
 
   For more details on how the weighting of the detector pixel fluxes are used in determining the final spaxel flux see
   the :ref:`weighting` section.
+
+A parameter only used for investigating which detector pixels contributed to a cube spaxel is ``debug_spaxel``. This option is only valid if the ``weighting`` parameter is set to ``drizzle`` (default). 
+
+``debug_spaxel [string]``
+
+  The string is the x,y,z value of the cube spaxel that is being investigated. The  numbering starts counting at 1. For example,
```

Comment:
Is the choice of 1-indexing for these values the best option? If someone is poking around at a cube in python, they'll have to convert from 0-indexed to 1-indexed to use this argument. I know that using 0 as the default value is convenient for detecting when there's no user input, but that could get changed to something like -1 if you switched to zero-indexed.

Reply:
The standard way to find these values is look in ds9 and find the values. So I was trying to make it is a simple as possible. But I can change it to 0 based to make it consistent with python.

Reply:
OK, if you think most folks are using ds9 to get the pixel indexes, then having them 1-indexed makes sense.